pages/git-protocol.md

```diff
@@ -21,7 +23,9 @@ A guide for programming within version control.
 * Perform work in a feature branch.
 * Rebase frequently to incorporate upstream changes.
 * Use a [pull request] for code reviews.
+* [Sign your commits with a GPG key](https://help.github.com/categories/gpg/).
```

Comment:
is this a requirement or a suggestion? (including text that explains the degree to which this is recommended would be helpful!)